src/main/g8/src/main/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
```

Comment:
Awesome! It would be great to add this to [twitter4s-demo](https://github.com/DanielaSfregola/twitter4s-demo) as well.

Created ticket for twitter4s-demo: https://github.com/DanielaSfregola/twitter4s-demo/issues/16

Reply:
Sure thing, will do !